FWCore/Framework/src/EventProcessor.cc

```diff
@@ -661,22 +624,6 @@ namespace edm {
     pathsAndConsumesOfModules.initialize(schedule_.get(), preg());
 
     std::vector<ModuleProcessName> consumedBySubProcesses;
```

Comment:
Is the `consumedBySubProcess` something to be cleaned up in a subsequent PR?

Reply:
Yes that is part of the module deletion code. I was going to tackle that next.